### PR TITLE
fix #1771: add romanian

### LINKED ISSUE #1771: Locale request: Romanian
- Wiktionary page: https://ro.wiktionary.org

I was contacted by email about potential Romanian dictionaries. So lets add them :)

@lasconic I'll give a try, or do you want to work on it?

### PR DESCRIPTION
**Pull request by lasconic**
Related to #1771

### CODE REVIEW

**Comment by BoboTiG** [comment]
Wicked! Thanks a lot!
I'll work on tests if you are short in time, no worries 👍🏻

File: wikidict/lang/ro/__init__.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+Words count: {words_count}
+Wiktionary dump: {dump_date}
+
+Available files:
+
+- [Kobo]({url_kobo}) (dicthtml-{locale}-{locale}.zip)
+- [StarDict]({url_stardict}) (dict-{locale}-{locale}.zip)
+- [DictFile]({url_dictfile}) (dict-{locale}-{locale}.df.bz2)
+
+<sub>Updated on {creation_date}</sub>
```
Comment (BoboTiG) [inline wikidict/lang/ro/__init__.py:57]:
```suggestion
Număr de cuvinte: {words_count}
Extragerea datelor din Wikționar: {dump_date}

Fișiere disponibile:

- [Kobo]({url_kobo}) (dicthtml-{locale}-{locale}.zip)
- [StarDict]({url_stardict}) (dict-{locale}-{locale}.zip)
- [DictFile]({url_dictfile}) (dict-{locale}-{locale}.df.bz2)

<sub>Ultima actualizare în {creation_date}</sub>
```

File: wikidict/lang/ro/__init__.py [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+"""  # noqa
+
+# Dictionary name that will be printed below each definition
+wiktionary = "Wiktionary (ɔ) {year}"
```
Comment (BoboTiG) [inline wikidict/lang/ro/__init__.py:61]:
```suggestion
wiktionary = "Wikționar (ɔ) {year}"
```

**Comment by lasconic** [comment]
* First etymology (done)
* Genre, more or less... a single word can be a lot of different parts of speech, and different genre: we try to get them all.
* Pronunciation. We get one for sure. I couldn't find a word with two different pronunciations.
* Definitions should be OK and readable since there is almost no templates (Format in romanian) used.

The translations are still in.

I would merge and release a version to get some feedback. Let's discuss variants later.

**Comment by BoboTiG** [comment]
Excellent 👍🏻

**Pull request merged**